sushy_oem_idrac/utils.py

```diff
@@ -38,3 +43,32 @@ def reboot_system(system):
         system.refresh()
 
     LOG.info('System powered ON')
+
+@retrying.retry(
+    retry_on_exception=lambda exception: isinstance(exception, Exception),
+    stop_max_attempt_number=_IDRAC_IS_READY_RETRIES,
+    wait_fixed=_IDRAC_IS_READY_RETRY_DELAY_SEC * 1000)
+def wait_for_idrac_ready(oem_manager):
+    is_ready_response = is_idrac_ready(oem_manager)
+    if "LCStatus" in is_ready_response:
+         LOG.info("idrac for node is ready")
+         return True
+    else:
+        err_msg = ('idrac for node is not ready,'
+                'Failed to perform drac operation, Retrying it' )
+        raise sushy.exception.ConnectionError(error=err_msg)
+
+def is_idrac_ready(oem_manager):
+
+    try:
+        response = asynchronous.http_call(
```

Comment:
Note that we need to convert http 400 errors into False

Reply:
Addressed, Taken care of http 400 errors code by converting it into False.